server/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "server",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "./app.js",
   "scripts": {
```

Comment:
Might be nice to have a script to start the server, e.g. start with node or dev with nodemon.